frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.css

```diff
@@ -2,7 +2,7 @@
 	display: flex;
 	gap: 1.5rem;
 	align-items: center;
-	padding: 0.5rem 1.5rem;
+	padding: 0 1.5rem;
```

Comment:
I think we leave this at 0.5rem top and bottom. On the smallest breakpoint I also knocked down the gap on this class from 1.5 to 1.25rem. Could possibly go down to 1rem, unsure.